ios/tooling/RIB.xctemplate/Default/___FILEBASENAME___Router.swift

```diff
@@ -18,7 +18,7 @@ final class ___VARIABLE_productName___Router: Router<___VARIABLE_productName___I
     // TODO: Constructor inject child builder protocols to allow building children.
```

Comment:
I think you would also have to remove the `override` keyword because you're not overriding a designated initializer from the superclass.